helm_chart/values.yaml

```diff
@@ -14,9 +14,12 @@ image:
 
 env: 
   auth_service: KeycloakAuthService
+  upload_service: Mock

```

Comment:
Do we want this defaulting to mock and not the hdfs direct data wave upload service? I suppose it won't really matter till we have somewhere to deploy to and it's easy enough to change.

Reply:
within the app itself, it does default to Mock. This is the helm chart so this is to set the value when deploying. I was using Mock because I was trying to get it connected to OPAL keycloak, with no success. I want to get that working before trying out the HDFS.